FEATURE_GATES.md

```diff
@@ -34,6 +34,14 @@ An additional **warning** for end-users who are reading this documentation and t
 [api-ref]:https://docs.konghq.com/kubernetes-ingress-controller/latest/references/custom-resources/
 [kic-guides]:https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/overview/
 
+### Documentation
+
+If you're looking for documentation for `Alpha` quality features you can find feature preview documentation [here in this repo](/FEATURE_PREVIEW_DOCUMENTATION.md).
```

Comment:
I don't like the "alpha quality" lingo.

Quality is a function of the process; alpha/beta/GA/etc. is a maturity stage.

Suggestion: drop the "alpha quality" language and think about something like "alpha maturity".

Reply:
```suggestion
If you're looking for documentation for `Alpha` maturity features you can find feature preview documentation [here in this repo](/FEATURE_PREVIEW_DOCUMENTATION.md).
```